docs/gitbook/reference/interbit-covenant-utils/destroy.md

```diff
@@ -17,7 +17,13 @@ An action that can be dispatched to a chain to destroy it.
 #### Example
```

Comment:
Why are we documenting `interbit-covenant-utils`? This functionality is exposed in `interbit-covenant-tools` and you have used tools in all your other examples. We have done everything we can to keep users from needing to work with `interbit-core` and `interbit-covenant-utils` directly and there are only 4 direct references in `package.json`.

Reply:
Fair, but can be addressed subsequently.